### PR TITLE
Added goldair tower fan config

### PR DESCRIPTION
**Pull request by horse-pants**
Tested with two fans at my home

### CODE REVIEW

**Comment by horse-pants** [comment]
The product name coming out of the tuya APIs is all upper case, so I copied it in the same because I wasn't sure if they had to match. 

If the casing doesn't matter I can make it normal casing

**Comment by horse-pants** [comment]
Ah true, I forgot about the temp sensor.

I'll add this tonight, thanks

File: custom_components/tuya_local/devices/goldair_platinum_tower_fan_with_wifi.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      name: oscillate
+      mapping:
+        - dps_val: "on"
+          value: true
+        - dps_val: "off"
+          value: false
+    - id: 6
+      type: integer
+      name: timer
+    - id: 19
```
Comment (make-all) [inline custom_components/tuya_local/devices/goldair_platinum_tower_fan_with_wifi.yaml:42]: I would make this a standalone sensor, as temperature is not part of the fan entity standard API.

Under secondary_entities:
```
  - entity: sensor
    class: temperature
    name: Temperature
    dps:
      - id: 19
        type: integer
        name: sensor
        class: measurement
        unit: C
```

Comment (horse-pants) [reply custom_components/tuya_local/devices/goldair_platinum_tower_fan_with_wifi.yaml]: Good call, I figured there had to be a way to make them their own sensor

Thanks

**Pull request merged**